tensorflow_io/azure/README.md

```diff
@@ -1,14 +1,19 @@
-# Azure storage integration
+# Azure blob storage integration
 
 ## Configuration via environment variables
-- `TF_AZURE_STORAGE_KEY`
-- `TF_AZURE_STORAGE_USE_HTTPS`
-- `TF_AZURE_STORAGE_BLOB_ENDPOINT`
+- `TF_AZURE_USE_DEV_STORAGE`  
+   Set to 1 to use local development storage emulator for connections like 'az://devstoreaccount1/container/file.txt'. This will take precendence over all other settings so `unset` to use any other connection
+- `TF_AZURE_STORAGE_KEY`  
+   Account key for the storage account in use
+- `TF_AZURE_STORAGE_USE_HTTP`  
+  Set to any value if you don't want to use https transfer. `unset` to use default of https 
+- `TF_AZURE_STORAGE_BLOB_ENDPOINT`  
+  Set to the endpoint of blob storage - default is .core.windows.net
 
 ```python
 >>> import os
 >>> import tensorflow as tf
->>> import tensorflow_io.azure.python.ops.azfs_ops
+>>> import tensorflow_io.azure.azfs_ops
```

Comment:
@damienpontifex I think we could move one step further, by moving `azfs_ops.py` to `__init__.py` (under the same directory). Then we could use:
```
import tensorflow_io.azure
```